src/components/User.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+export default class User extends Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components